activestorage/lib/active_storage/service/configurator.rb

```diff
@@ -32,5 +32,16 @@ def resolve(class_name)
       rescue LoadError
         raise "Missing service adapter for #{class_name.inspect}"
       end
+
+      def build_configurations(configurations)
+        if storage_url = ENV["STORAGE_URL"]
```

Comment:
Is this supposed to be comparison or is it an assignment here?

Reply:
This is a conditional assignment, in order to give `ENV["STORAGE_URL"]` priority over the rest of the configuration.

Active Storage doesn't need to support multiple configurations if the URL is set, unlike AR where you can have 2 or 3-tierd configs, so this is a simpler approach.